### PR TITLE
Create Changelogs “4-11-1-2020-01-13”

### PR DESCRIPTION
**Pull request by alioso**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/content/changelog/4-11-1-2019-12-23.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: '4.11.1: 2020-01-13'
+date: 2020-01-13T05:00:00.000Z
+version: 4.11.1
+changelog:
+  - API
+---
+_Fixes_
```
Comment (Jskobos) [inline src/content/changelog/4-11-1-2019-12-23.md]: This is italicized in the output, which doesn't match the style of the earlier entries.

File: src/content/changelog/4-11-1-2019-12-23.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: '4.11.1: 2020-01-13'
```
Comment (Jskobos) [inline src/content/changelog/4-11-1-2019-12-23.md:2]: Is this the correct date?

**Review by Jskobos — APPROVED** [review]

File: src/content/changelog/4-11-1-2019-12-23.md (hunk 2 above)
Comment (nmelehan) [reply src/content/changelog/4-11-1-2019-12-23.md:2]: yes--we previously were going to release a hotfix earlier, but then decided not to, and we kept this PR to store the changes for the Agnostic Front release this coming monday

File: src/content/changelog/4-11-1-2019-12-23.md (hunk 1 above)
Comment (nmelehan) [reply src/content/changelog/4-11-1-2019-12-23.md]: Fixed in latest commit

**Review by alsaihn — APPROVED** [review]

**Pull request merged**